ntbs-service/Pages/Notifications/Overview.cshtml

```diff
@@ -15,7 +15,7 @@
 
 <partial name="./OverviewPartials/_ClinicalDetailsOverviewPartial" />
 
-<partial name="./OverviewPartials/_ContactTracingOverviewPartial" for="Notification.ContactTracing" />
+<partial name="./OverviewPartials/_ContactTracingOverviewPartial" />

```

Comment:
It's a bit of a shame that the partials can no longer be scoped to the entity they cover...
Would it make sense to pull the `edit` links out to this file? Perhaps we could actually really cut down on the repetition by introducing a loop over the partials' paths?

Reply:
I thought about that - it was partially because we were doing a mix of the overviewModel and entity models throughout this file, i thought about pulling the edit links out but couldn't get a clean solution yet